src/Azure.Functions.Cli/Helpers/DotnetHelpers.cs

```diff
@@ -86,6 +86,7 @@ public static async Task DeployDotnetFunction(string templateName, string functi
         private static string GetTemplateShortName(string templateName) => templateName.ToLowerInvariant() switch
```

Comment:
Tested locally and looks good to me. We'll just need to update the nupkg versions once those are out. Example here: https://github.com/Azure/azure-functions-core-tools/pull/3493

Reply:
Updated to 4.0.3038